# File where data will be stored
filename: "test_optoforce_node"

# Maximun number of samples to be stored
num_samples: 120000

# Publish frequency
loop_rate: 100

# Desired devices to connect
connectedDAQs: 2

# OptoForce Sensors to be opened
devices:

  - name: "IRE005"                                        # Serial Number of the Device. IRE005 Blue Box
    calibration: [92.6,93.6,20.12,5054.3,5085.4,6912.5]   # IRE005 Calibration from Sensitivity Report
    speed: 1000                                           # Transmission frequency in Hz
    force_transformation: [ 0, 1, 0, -1, 0, 0, 0, 0, -1]
    torque_transformation: [ 0, 1, 0, 1, 0, 0, 0, 0, 1]

  - name: "IRE004"                                          # Serial Number of the Device. IRE004 Black Box
    calibration: [97.78,101.72,20.53,5210.6,5267.2,7659.7]  # IRE004 Calibration from Sensitivity Report
    speed: 1000                                             # Transmission frequency in Hz
    force_transformation: [ 0, -1, 0, -1, 0, 0, 0, 0, 1]
    torque_transformation: [ 0, -1, 0, 1, 0, 0, 0, 0, 1]
